runbotics/runbotics-orchestrator-ui/src/src-app/components/BotProcessRunner.tsx

```diff
@@ -44,54 +69,53 @@ interface BotProcessRunnerProps {
 const BotProcessRunner: FC<BotProcessRunnerProps> = ({
     className,
     process,
+    externalProcessInstance,
+    isProcessInstanceActive,
```

Comment:
if the process instance is active you can look up the `active` object in redux. Whether it exists and has status finished it means that smthing is processing right now

Reply:
I see that you do it already so its surely redundant prop

Reply:
I've changed it to a function "setInitialState" used inside the BotProcessRunner, is it acceptable? 👯